src/cli/__snapshots__/format.int.test.ts.snap

```diff
@@ -118,7 +118,7 @@ Processed 2 files in <random>s.
   8:3  error  Unexpected console statement  no-console
 
 <random>/d.js
-  1:0  error  Parsing error: Declaration or statement expected
+  1:0  error  Parsing error: Unexpected token (1:0)
```

Comment:
The old error was reported by `typescript-eslint`. This new error is reported by ESLint directly.